requirements.txt

```diff
@@ -1,141 +1,144 @@
 -i https://pypi.org/simple
-aiohttp==3.8.4 ; python_version >= '3.6'
+aiohttp==3.8.5 ; python_version >= '3.6'
 aiosignal==1.3.1 ; python_version >= '3.7'
-ape-solidity==0.6.8
+ape-etherscan==0.6.10
+ape-polygon==0.6.5
+ape-solidity==0.6.9
```

Comment:
`ape-infura` should be here

Reply:
Or rather, it should be in the Pipfile (and hence here as well)

Reply:
Yes. I intend to add it to the Pipfile and do a relock. The tricky thing at the moment is that eth-ape 0.6.20 (latest version) has a bug that causes our tests to fail.

So I may relock with a limit on the eth-ape version to be <=0.6.19

Reply:
✔️ 

If you are wondering about the test failure:

```bash

 [22] → pip install -U eth-ape
 ...
 Successfully installed eth-ape-0.6.20 urllib3-1.26.16
 

 [23] → ape test tests/test_coordinator.py::test_initiate_ritual
==================================================== test session starts ====================================================
platform darwin -- Python 3.9.17, pytest-7.4.2, pluggy-1.3.0
rootdir: /Users/derek/Documents/Github/repos/forks/derek/nucypher-contracts
plugins: eth-ape-0.6.20, web3-6.10.0
collected 1 item

tests/test_coordinator.py F                                                                                           [100%]

========================================================= FAILURES ==========================================================
___________________________________________________ test_initiate_ritual ____________________________________________________
/Users/derek/Documents/Github/repos/forks/derek/nucypher-contracts/tests/test_coordinator.py:184: in test_initiate_ritual
    assert event["participants"] == tuple(n.address.lower() for n in nodes)
E   AssertionError: assert [['0x1e14cbd1...d628176a1e7']] == ('0x1e14cbd14...5d628176a1e7')
E     At index 0 diff: ['0x1e14cbd146c3a5496f588cf28ee656c210057904', '0x1e59ce931b4cfea3fe4b875411e280e173cb7a9c', '0x318b469bba396aec2c60342f9441be36a1945174', '0xc89d42189f0450c2b2c3c61f58ec5d628176a1e7'] != '0x1e14cbd146c3a5496f588cf28ee656c210057904'
E     Right contains 3 more items, first extra item: '0x1e59ce931b4cfea3fe4b875411e280e173cb7a9c'

```

For some reason the "participants" entry in the `StartRitual` event is interpreted as a list of lists instead of just a list. This does not happen in 0.6.19.

Reply:
FYI reached out to the ape team about this and they are looking into it - https://discord.com/channels/922917176040640612/938897808147247186/1156375065156391042.

Reply:
also I'll have a look into it, because that assert is old style event assert and in 0.6.20 they introduced fix for structs in events so new style should work better